homeassistant/components/unifi_direct/manifest.json

```diff
@@ -1,9 +1,8 @@
 {
   "domain": "unifi_direct",
   "name": "UniFi AP",
-  "codeowners": [],
+  "codeowners": ["@tofuSCHNITZEL"],
   "documentation": "https://www.home-assistant.io/integrations/unifi_direct",
   "iot_class": "local_polling",
-  "loggers": ["pexpect", "ptyprocess"],
-  "requirements": ["pexpect==4.6.0"]
+  "requirements": ["unifi_ap==0.0.1"]
```

Comment:
Shouldn't you add your lib to the loggers list?

Reply:
my lib does not (currently) use the logging package or getLogger - should I still add it here?

Reply:
I think it's useful. It's something that you will forget later on and then if you ever need debug logging you don't get any. Can be frustrating to find out afterwards.

Reply:
yes makes sense!